docusaurus.config.js

```diff
@@ -6,84 +6,27 @@ module.exports = {
   tagline: 'The Libra Association’s mission is to enable a simple global payment system and financial infrastructure that empowers billions of people.',
   url: 'https://your-docusaurus-test-site.com',
   baseUrl: '/',
-  favicon: 'img/favicon.ico',
-  organizationName: 'facebook', // Usually your GitHub org/user name.
-  projectName: 'docusaurus', // Usually your repo name.
+  favicon: 'img/shared/favicon.ico',
+  organizationName: 'Libra',
+  projectName: 'Libra',
   themeConfig: {
     algolia: {
       apiKey: 'eb29b473d27eae9cc46c84eb3a2e4063',
       indexName: 'libra-website',
     },
+    prism: {
+      darkTheme: darkCodeTheme,
+      theme: lightCodeTheme,
+      enableLineNumbers: true,
+    },
```

Comment:
Thought this was in a previous commit (The series that added the code snippet). Weird